static/js/services/changes.js

```diff
@@ -57,7 +60,9 @@ angular.module('inboxServices').factory('Changes',
         });
     };
 
-    watchChanges();
+    DB().info().then(function(info) {
+      lastSeq = info.update_seq;
+    }).then(watchChanges);
```

Comment:
It's the equivalent, with the added bonus that we get to store the actual value in `lastSeq`. The reasoning being that it solves this bug without requiring at least one successful watch to occur. FWIW I don't think this actually ads a new request, because the `now` is resolved client-side by PouchDB by doing the exact same thing that I'm doing (requesting info and then specifying the exact rev)